flexget/plugins/filter/exists_movie.py

```diff
@@ -162,28 +165,36 @@ def on_task_filter(self, task, config):
             count_entries += 1
             log.debug('trying to parse entry %s' % entry['title'])
             if config.get('lookup') == 'imdb':
-                key = 'imdb_id'
-                if not entry.get('imdb_id', eval_lazy=False):
+                keyImdb = 'imdb_id'
```

Comment:
please stick to snake_case. Also, if this isn't used anywhere else, I really don't see the point of assigning into variable.